ada_url/ada_adapter.py

```diff
@@ -290,7 +320,9 @@ def normalize_url(s):
     return parse_url(s, attributes=('href',))['href']
 
 
-def parse_url(s, attributes=PARSE_ATTRIBUTES):
+# FIXME constrain `attributes``
```

Comment:
```suggestion
# FIXME constrain `attributes`
```

Reply:
Same here - the docstring says unrecognized requests are ignored; we need not constrain with the type if we preserve that behavior.